CMakeLists.txt

```diff
@@ -481,6 +481,26 @@ if(BUILD_BINDINGS_JAVA OR BUILD_BINDINGS_PYTHON)
   add_subdirectory(src/bindings)
 endif()
 
+#-----------------------------------------------------------------------------#
+# Package configuration
+#
+# Install config and configversion for CVC4 for find_package(CVC4).
+
+include(CMakePackageConfigHelpers)
+write_basic_package_version_file(
+  ${CMAKE_CURRENT_BINARY_DIR}/CVC4ConfigVersion.cmake
+  VERSION ${CVC4_RELEASE_STRING}
+  COMPATIBILITY SameMajorVersion
```

Comment:
Hmm, isn't it more like `SameMinorVersion` for us? E.g. for 1.7 we [changed the API a bit](https://github.com/CVC4/CVC4/blob/eba03c5af0112aea04d83977333ae37e8a13137d/NEWS#L39-L40).

Reply:
Yep, changed.